readthedocs/rtd_tests/utils.py

```diff
@@ -34,6 +34,11 @@ def make_test_git():
     log.info(check_output(['git', 'init'] + [directory], env=env))
     log.info(check_output(['git', 'add', '.'], env=env))
     log.info(check_output(['git', 'commit', '-m"init"'], env=env))
+    # Add repo itself as submodule
+    log.info(check_output(['git', 'checkout', '-b', 'submodule'], env=env))
+    log.info(check_output(['git', 'submodule', 'add', '-b', 'master', './', 'submodule'], env=env))
```

Comment:
I added the repo itself as submodule on other branch for test the submodule existence

Reply:
Isn't it needed to re-checkout master after creating the submodule and adding it? I mean, so other tests keep using the `master` branch as they did before.